components/sup/src/main.rs

```diff
@@ -114,26 +173,31 @@ fn cli<'a, 'b>() -> App<'a, 'b> {
                 [default: default]")
             (@arg PEER: --peer +takes_value +multiple
                 "The listen address of an initial peer (IP[:PORT])")
-            (@arg PERMANENT_PEER: --("permanent-peer") -I "If this service is a permanent peer")
+            (@arg PERMANENT_PEER: --("permanent-peer") -I "If this Supervisor is a permanent peer")
```

Comment:
Much better and more correct. The previous description wasn't even technically true.